docs/changes/0.27.0.rst

```diff
@@ -1,35 +1,67 @@
 Changelog for QCoDeS 0.27.0
 ===========================
 
-The June 2021 release of QCoDeS.
+The July 2021 release of QCoDeS.
 
 -----------------
 Breaking Changes:
 -----------------
 
-- DelegateInstrument takes `parameters` as an optional, keyworded input argument. Any arguments passed as positional arguments after this one will break with this change (#3089).
+- DelegateInstrument takes `parameters` as an optional, keyworded input argument. Any arguments passed as positional arguments after this one will break with this change. (#3089)
 
 ----
 New:
 ----
 
 - The default value for `default_parameter_name` in the Loop QCoDeS dataset changed from ``amplitude`` to ``None``.
   The ``amplitude`` value interferes with datasets where one of the setpoint arrays is called `amplitude`. (#3095)
+- Abstract parameters has been added which allow one to create abstract instrument types that are guaranteed 
+to have certain parameters present. (#3125)
```

Comment:
```suggestion
- Abstract parameters has been added which allow one to create abstract instrument types that have parameters which can be overwritten in a subclass allowing the creation of instruments subclasses with a well-defined api. (#3125)
```

Reply:
Since the part about checking for abstracparameters got reverted lets rephrase this

Reply:
sound good, thanks Jens